delivery_recipe/delivery_team.md

```diff
@@ -0,0 +1,86 @@
+
+The Service Delivery Director (SDD) has experience of change and strategy to keep the client ahead of the change programme, to plan further ahead, to identify opportunities and to discuss these with stakeholders. Discussing these ahead of time keeps the business one step ahead, providing stakeholders with the information to make informed decisions about steering the business and aligning the different departments.
+
+On a different level, in support of the delivery team, the SDD will also provide coaching and support to the Product Owner and the Delivery Team. Being less connected to the day-to-day activities they can view the work from a different perspective, offer alternative solutions and provide different thinking that can question decisions being made. This is not intended to contradict the team, only to confirm their thinking and decisions.
+
+### Service Delivery Manager
+
+The Service Delivery Manager (SDM) is the conductor in the change orchestra. Their job is to:
+
+* support the Product Owner
+* support the Delivery Team
+* lead meetings
+* keep the delivery on track
+* produce the management reports
+* present progress to the steering group
+* facilitate communication within the delivery team
+* communicate progress and plans to the wider business
+
+The SDM is not a project manager, they’re more akin to a coach. The SDM provides the support to the delivery team to keep the delivery running smooth while supporting the Product Owner to keep them ahead of what’s being delivered now so they can plan and prepare in advance.
+
+### Lead Developer
+
+The Lead Developer is a senior role within the team. They are not the team leader but they are looked up to by other members of the delivery team because of their experience, skills, their ability to dissect a problem and their understanding of how an approach to a requirement will affect the wider solution.
+
+The key aspects of this role are:
+
+* define the overall architecture
+* be able to make technical decisions for the team
+* support and coach other members of the delivery team
+* build client trust through confidence and knowledge
+
+Ultimately, the Lead Developer is the person who will make, and stand by, the technical decisions for the team.
+
+### Lead UX Designer
+
+Whatever the project or change programme, the ultimate goal is to provide or make things better for the user. The job of the Lead UX Designer is to align the form and the function of what the delivery team develop with the needs of the user.
+
+To protect the acceptability and longevity of what we’re pouring money, effort and time into it’s the UX Designer’s job to:
+
+* Speak to users, to find out what they need
+* Monitor the interaction of users as they get hands-on with what we’re building
+* Turn interviews, analytics, feedback and observations into requirements for the team
```

Comment:
For the specific details around user research, usability and design concepts, lets include these as separate pages in the actual delivery/process pages.